tests/ipfwd/test_nhop_group.py

```diff
@@ -482,16 +484,34 @@ def test_nhop_group_member_order_capability(duthost, tbinfo, ptfadapter, gather_
     # create nexthop group
     nhop = IPRoutes(duthost, asic)
 
-    recvd_pkt_result = defaultdict(int)
+    recvd_pkt_result = defaultdict(set)
 
     rtr_mac = asic.get_router_mac()
 
-    pkt, exp_pkt = build_pkt(rtr_mac, ip_route, ip_ttl)
 
-    for x in range(2):
+    def built_and_send_tcp_ip_packet():
+        for flow_count in range(50):
+            pkt, exp_pkt = build_pkt(rtr_mac, ip_route, ip_ttl, flow_count)
+            testutils.send(ptfadapter, gather_facts['dst_port_ids'][0], pkt, 10)
+            (_, recv_pkt) = testutils.verify_packet_any_port(test=ptfadapter, pkt=exp_pkt,
+                                                             ports=gather_facts['src_port_ids'])
+
+            assert recv_pkt
+
+            # Make sure routing is done
+            pytest_assert(scapy.Ether(recv_pkt).ttl == (ip_ttl - 1), "Routed Packet TTL not decremented")
+            pytest_assert(scapy.Ether(recv_pkt).src == rtr_mac, "Routed Packet Source Mac is not router MAC")
+            pytest_assert(scapy.Ether(recv_pkt).dst.lower() in neighbor_mac,
+                          "Routed Packet Destination Mac not valid neighbor entry")
+
+            recvd_pkt_result[flow_count].add(scapy.Ether(recv_pkt).dst)
+
+    # Test/Iteration Scenario 1: Verify After ecmp member remove/add flow order remains same.
```

Comment:
Is it true that the flow order remains the same after a new meber is added or removed?

Reply:
this scenario verifies link down and up scenario. In my case when link down happens we remove all member from the group and then add it back when link comes back up. In such case yes flow order remain same.